From: Carlos Garnacho <carlosg@gnome.org>
Date: Fri, 31 Jul 2020 20:15:44 +0200
Subject: backends/native: Drop early processing of ClutterEvents

We have 2 sources (this one in MetaSeatNative, and the one in
MetaBackend) dispatching ClutterEvents to the stage. Make the
MetaSeatNative one exclusively about dispatching the libinput
queue, and leave ClutterEvents to the other.

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1403>
---
 src/backends/native/meta-seat-native.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/backends/native/meta-seat-native.c b/src/backends/native/meta-seat-native.c
index 783d9d1..0c6b0a3 100644
--- a/src/backends/native/meta-seat-native.c
+++ b/src/backends/native/meta-seat-native.c
@@ -1347,13 +1347,6 @@ meta_event_dispatch (GSource     *g_source,
   dispatch_libinput (seat);
 
  queue_event:
-  event = clutter_event_get ();
-
-  if (event)
-    {
-      /* forward the event into clutter for emission etc. */
-      _clutter_stage_queue_event (event->any.stage, event, FALSE);
-    }
 
   return TRUE;
 }
